src/test/java/org/takes/facets/auth/PsBasicTest.java

```diff
@@ -185,11 +189,59 @@ public void handleMultipleHeadersWithInvalidContent() throws Exception {
     }
 
     /**
+     * Test {@link org.takes.facets.auth.PsBasic.Default#enter} retunrs
+     * a valid user.
+     * With user and pass ok.
+     */
+    @Test
+    public void returnsCorrectUrn() {
+        final String user = "Admin";
+        final String pass = "123";
+        final String urn = "urn:basic:admin";
+        final List<User> usrs =
+            new ArrayList<User>(1);
+        usrs.add(new User(user, pass, urn));
+        final Opt<Identity> identity = new PsBasic.Default(usrs)
+            .enter(user, pass);
+        MatcherAssert.assertThat(identity.has(), Matchers.is(true));
+        MatcherAssert.assertThat(
+            identity.get().urn(),
+            CoreMatchers.equalTo(urn)
+        );
+    }
+
+    /**
+     * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
+     * With user ok and pass wrong returns empty identity.
+     */
+    @Test
+    public void returnsFalseIfUserDoesNotExist() {
+        final Opt<Identity> identity =
+            new PsBasic.Default("Adm,123,urn:basic:adm").enter("Adm", "125");
+        MatcherAssert.assertThat(identity.has(), Matchers.is(false));
+    }
+
+    /**
+     * Test {@link org.takes.facets.auth.PsBasic.Default#enter}.
+     * With user wrong and pass ok returns empty indentity.
+     */
+    @Test
+    public void returnsFalseForWrongPassword() {
+        final String pwd = "126";
+        final List<User> usrs =
+                new ArrayList<User>(1);
+        usrs.add(new User("Adminis", pwd, "urn:basic;admins"));
+        final Opt<Identity> identity = new PsBasic.Default(usrs)
```

Comment:
@lautarobock `identity` looks like one-time variable for me, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables